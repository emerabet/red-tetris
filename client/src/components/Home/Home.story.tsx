import React from 'react';
import { storiesOf } from '@storybook/react';
import Home from './Home';

storiesOf('Home', module)
    .add('empty', () => (
        <Home
            room=""
            player=""
            enterRoom={(_) => { return 'void'; }}
            handleChange={(_) => { return 'void'; }}
        />
    ))
    .add('full', () => (
        <Home
            room="my room"
            player="my name"
            enterRoom={(_) => { return 'void'; }}
            handleChange={(_) => { return 'void'; }}
        />
    ));
