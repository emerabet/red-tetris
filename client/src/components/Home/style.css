.mainDivHome {
    min-height: 100vh;
    font-size: calc(2px + 2vmin);
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}

.formFlex {
  margin-top: 80px;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
  }

.namer {
    position: relative;
    margin: 5px auto 0;
   }
   
   .namer input {
    border: 0;
    border-bottom: 2px solid #1976D2;
    width: 100%;
    font-size: 15px;
    text-align: center;
    padding: 5px;
    background: transparent;
       color: #BBDEFB;
   }
   
   .namer input.shake {
    animation-name: shaker;
    animation-duration: 200ms;
    animation-timing-function: ease-in-out;
    animation-delay: 0s;
   }
   
   .namer input:focus {
    outline: 0;
       color: rgb(227, 233, 140);
   }
   
   .namer input::placeholder {
    color: #1976D2;
   }
   
   
   
   
   
   @keyframes shaker {
    0% {
     transform: translate(0px, 0px) rotate(0deg);
     opacity: 0.8;
    }
    10% {
     transform: translate(10px, 7px) rotate(-9deg);
     opacity: 0.6;
    }
    20% {
     transform: translate(13px, -19px) rotate(-3deg);
     opacity: 0.3;
    }
    30% {
     transform: translate(-6px, -6px) rotate(2deg);
     opacity: 0.4;
    }
    40% {
     transform: translate(-9px, -18px) rotate(-5deg);
     opacity: 0.4;
    }
    50% {
     transform: translate(10px, -8px) rotate(5deg);
     opacity: 0.7;
    }
    60% {
     transform: translate(-10px, 14px) rotate(-6deg);
     opacity: 1;
    }
    70% {
     transform: translate(10px, 3px) rotate(6deg);
     opacity: 0.1;
    }
    80% {
     transform: translate(-2px, 20px) rotate(-6deg);
     opacity: 1;
    }
    90% {
     transform: translate(-7px, -19px) rotate(2deg);
     opacity: 0.5;
    }
   }


   /* BUTTON PLAY */

@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro:300);

.plus {
  margin-top: 60px;
  position: relative;
  z-index: 1;
  width: 120px;
  height: 40px;
  color: #1a1a1a;
  background: #ed1c5b;
  border-radius: 5px;
  border: none;
  transition: box-shadow 400ms cubic-bezier(0.2, 0, 0.7, 1), -webkit-transform 200ms cubic-bezier(0.2, 0, 0.7, 1);
  transition: box-shadow 400ms cubic-bezier(0.2, 0, 0.7, 1), transform 200ms cubic-bezier(0.2, 0, 0.7, 1);
  transition: box-shadow 400ms cubic-bezier(0.2, 0, 0.7, 1), transform 200ms cubic-bezier(0.2, 0, 0.7, 1), -webkit-transform 200ms cubic-bezier(0.2, 0, 0.7, 1);
}
.plus:after {
  content: "JOIN";
  font-size: 2.5em;
  line-height: 1.1em;
  color: #f8b500;
}

.plus:hover {
  box-shadow: 0 0 1px 15px rgba(138, 59, 88, 0.4), 0 0 1px 30px rgba(138, 59, 88, 0.1), 0 0 1px 45px rgba(138, 59, 88, 0.1);
}

/* input {
  width: 120px;
  padding: 5px;
  border-radius: 5px;
} */

label {
  margin-top: 10px;
  margin-bottom: 2px;
}