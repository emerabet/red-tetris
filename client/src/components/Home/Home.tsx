import React, { Component, useState, useEffect } from 'react';
import SquareLabel from '../Labels';
import RedTetris from '../RedTetris';
import { useWindowSize } from '../../effects/useWindowSize';
import './style.css';

interface HomeProps {
  room: string;
  player: string;
  enterRoom: (event: any) => {};
  handleChange: (event: any) => void;
}

const Home: React.SFC<HomeProps> = (props) => {
  const [roomClassName, setRoomClassName] = useState('');
  const [playerClassName, setPlayerClassName] = useState('');
  const dim = useWindowSize();

  function roomShake() {
    setRoomClassName('shake');
    const t = setTimeout(
      () => {
        setRoomClassName('');
      },
      1000);
    return function clearUp() {
      clearTimeout(t);
    };
  }

  function playerShake() {
    setPlayerClassName('shake');
    const t = setTimeout(
      () => {
        setPlayerClassName('');
      },
      1000);
    return function clearUp() {
      clearTimeout(t);
    };
  }

  return (
    <div className="mainDivHome">
    <RedTetris additionalClassName={dim.outerWidth > 480 ? '' :
    dim.outerWidth > 340 ? 'medium' : 'small' }/>
      <form className="formFlex" onSubmit={props.enterRoom}>
        {props.room !== '' && <SquareLabel label="room" additionalClassName="big" />}
        <div className="namer">
          <div className="namer-input">
            <input
              onBlur={roomShake}
              className={`${roomClassName} roomInput`}
              name="room"
              type="text"
              placeholder="ROOM"
              value={props.room}
              onChange={props.handleChange}></input>
          </div>
        </div>
        {props.player !== '' && <SquareLabel label="player" additionalClassName="big" />}
        <div className="namer">
          <div className="namer-input">
            <input
              onBlur={playerShake}
              className={`${playerClassName} playerInput`}
              name="player"
              type="text"
              placeholder="PLAYER"
              value={props.player}
              onChange={props.handleChange}></input>
          </div>
        </div>
        <button
          type="submit"
          className="plus"
          disabled={props.room === '' || props.player === ''}></button>
      </form>
    </div>
  );
};

export default Home;
