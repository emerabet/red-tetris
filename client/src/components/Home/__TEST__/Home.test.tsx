import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer, { act } from 'react-test-renderer';
import Home from '../Home';
import 'react-testing-library/cleanup-after-each';

configure({ adapter: new Adapter() });

it('renders correctly with defaults', () => {
  const home = renderer.create(<Home
    room="my room"
    player="my name"
    enterRoom={(_) => { return 'void'; }}
    handleChange={(_) => { return 'void'; }}
  />).toJSON();
  expect(home).toMatchSnapshot();
});

it('renders correctly with defaults', () => {
  const home = renderer.create(<Home
    room=""
    player=""
    enterRoom={(_) => { return 'void'; }}
    handleChange={(_) => { return 'void'; }}
  />).toJSON();
  expect(home).toMatchSnapshot();
});

it('Test click event', () => {
  const mockCallBack = jest.fn();

  const wrapper = shallow((<Home
    room="test"
    player="test"
    enterRoom={mockCallBack}
    handleChange={(_) => { return 'void'; }}
  />));
  act(() => {
    global.outerWidth = 200;
    global.dispatchEvent(new Event('resize'));
  });
  act(() => {
    global.outerWidth = 480;
    global.dispatchEvent(new Event('resize'));
  });
  expect(wrapper.prop('onSubmit') === mockCallBack);
});
