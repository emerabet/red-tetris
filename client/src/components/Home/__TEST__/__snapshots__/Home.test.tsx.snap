// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<div
  className="mainDivHome"
>
  <div
    className="redTetris "
  >
    <div
      className="box boxR"
    >
      <div
        className="boxCube boxR1"
      />
      <div
        className="boxCube boxR2"
      />
      <div
        className="boxCube boxR3"
      />
      <div
        className="boxCube boxR4"
      />
      <div
        className="boxCube boxR5"
      />
      <div
        className="boxCube boxR6"
      />
      <div
        className="boxCube boxR7"
      />
      <div
        className="boxCube boxR8"
      />
      <div
        className="boxCube boxR9"
      />
      <div
        className="boxCube boxR10"
      />
      <div
        className="boxCube boxR11"
      />
      <div
        className="boxCube boxR12"
      />
      <div
        className="boxCube boxR13"
      />
      <div
        className="boxCube boxR14"
      />
    </div>
    <div
      className="box boxE"
    >
      <div
        className="boxCube boxE1"
      />
      <div
        className="boxCube boxE2"
      />
      <div
        className="boxCube boxE3"
      />
      <div
        className="boxCube boxE4"
      />
      <div
        className="boxCube boxE5"
      />
      <div
        className="boxCube boxE6"
      />
      <div
        className="boxCube boxE7"
      />
      <div
        className="boxCube boxE8"
      />
      <div
        className="boxCube boxE9"
      />
      <div
        className="boxCube boxE10"
      />
      <div
        className="boxCube boxE11"
      />
      <div
        className="boxCube boxE12"
      />
      <div
        className="boxCube boxE13"
      />
    </div>
    <div
      className="box boxD"
    >
      <div
        className="boxCube boxD1"
      />
      <div
        className="boxCube boxD2"
      />
      <div
        className="boxCube boxD3"
      />
      <div
        className="boxCube boxD4"
      />
      <div
        className="boxCube boxD5"
      />
      <div
        className="boxCube boxD6"
      />
      <div
        className="boxCube boxD7"
      />
      <div
        className="boxCube boxD8"
      />
      <div
        className="boxCube boxD9"
      />
      <div
        className="boxCube boxD10"
      />
      <div
        className="boxCube boxD11"
      />
      <div
        className="boxCube boxD12"
      />
    </div>
    <div
      className="titleSpace"
    />
    <div
      className="boxs boxT"
    >
      <div
        className="boxCube boxT1"
      />
      <div
        className="boxCube boxT2"
      />
      <div
        className="boxCube boxT3"
      />
      <div
        className="boxCube boxT4"
      />
      <div
        className="boxCube boxT5"
      />
      <div
        className="boxCube boxT6"
      />
      <div
        className="boxCube boxT7"
      />
    </div>
    <div
      className="box boxE"
    >
      <div
        className="boxCube boxE1"
      />
      <div
        className="boxCube boxE2"
      />
      <div
        className="boxCube boxE3"
      />
      <div
        className="boxCube boxE4"
      />
      <div
        className="boxCube boxE5"
      />
      <div
        className="boxCube boxE6"
      />
      <div
        className="boxCube boxE7"
      />
      <div
        className="boxCube boxE8"
      />
      <div
        className="boxCube boxE9"
      />
      <div
        className="boxCube boxE10"
      />
      <div
        className="boxCube boxE11"
      />
      <div
        className="boxCube boxE12"
      />
      <div
        className="boxCube boxE13"
      />
    </div>
    <div
      className="boxs boxT"
    >
      <div
        className="boxCube boxT1"
      />
      <div
        className="boxCube boxT2"
      />
      <div
        className="boxCube boxT3"
      />
      <div
        className="boxCube boxT4"
      />
      <div
        className="boxCube boxT5"
      />
      <div
        className="boxCube boxT6"
      />
      <div
        className="boxCube boxT7"
      />
    </div>
    <div
      className="box boxR"
    >
      <div
        className="boxCube boxR1"
      />
      <div
        className="boxCube boxR2"
      />
      <div
        className="boxCube boxR3"
      />
      <div
        className="boxCube boxR4"
      />
      <div
        className="boxCube boxR5"
      />
      <div
        className="boxCube boxR6"
      />
      <div
        className="boxCube boxR7"
      />
      <div
        className="boxCube boxR8"
      />
      <div
        className="boxCube boxR9"
      />
      <div
        className="boxCube boxR10"
      />
      <div
        className="boxCube boxR11"
      />
      <div
        className="boxCube boxR12"
      />
      <div
        className="boxCube boxR13"
      />
      <div
        className="boxCube boxR14"
      />
    </div>
    <div
      className="boxs boxI"
    >
      <div
        className="boxCube boxI1"
      />
      <div
        className="boxCube boxI2"
      />
      <div
        className="boxCube boxI3"
      />
      <div
        className="boxCube boxI4"
      />
      <div
        className="boxCube boxI5"
      />
      <div
        className="boxCube boxI6"
      />
      <div
        className="boxCube boxI7"
      />
      <div
        className="boxCube boxI8"
      />
      <div
        className="boxCube boxI9"
      />
    </div>
    <div
      className="box boxS"
    >
      <div
        className="boxCube boxS1"
      />
      <div
        className="boxCube boxS2"
      />
      <div
        className="boxCube boxS3"
      />
      <div
        className="boxCube boxS4"
      />
      <div
        className="boxCube boxS5"
      />
      <div
        className="boxCube boxS6"
      />
      <div
        className="boxCube boxS7"
      />
      <div
        className="boxCube boxS8"
      />
      <div
        className="boxCube boxS9"
      />
      <div
        className="boxCube boxS10"
      />
      <div
        className="boxCube boxS11"
      />
      <div
        className="boxCube boxS12"
      />
      <div
        className="boxCube boxS13"
      />
      <div
        className="boxCube boxS14"
      />
    </div>
  </div>
  <form
    className="formFlex"
    onSubmit={[Function]}
  >
    <div
      className="labelTitle big"
    >
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube r1"
          />
          <div
            className="labelCube r2"
          />
          <div
            className="labelCube r3"
          />
          <div
            className="labelCube r4"
          />
          <div
            className="labelCube r5"
          />
          <div
            className="labelCube r6"
          />
          <div
            className="labelCube r7"
          />
          <div
            className="labelCube r8"
          />
          <div
            className="labelCube r9"
          />
          <div
            className="labelCube r10"
          />
          <div
            className="labelCube r11"
          />
          <div
            className="labelCube r12"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube o1"
          />
          <div
            className="labelCube o2"
          />
          <div
            className="labelCube o3"
          />
          <div
            className="labelCube o4"
          />
          <div
            className="labelCube o5"
          />
          <div
            className="labelCube o6"
          />
          <div
            className="labelCube o7"
          />
          <div
            className="labelCube o8"
          />
          <div
            className="labelCube o9"
          />
          <div
            className="labelCube o10"
          />
          <div
            className="labelCube o11"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube o1"
          />
          <div
            className="labelCube o2"
          />
          <div
            className="labelCube o3"
          />
          <div
            className="labelCube o4"
          />
          <div
            className="labelCube o5"
          />
          <div
            className="labelCube o6"
          />
          <div
            className="labelCube o7"
          />
          <div
            className="labelCube o8"
          />
          <div
            className="labelCube o9"
          />
          <div
            className="labelCube o10"
          />
          <div
            className="labelCube o11"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetter"
        >
          <div
            className="labelCube m1"
          />
          <div
            className="labelCube m2"
          />
          <div
            className="labelCube m3"
          />
          <div
            className="labelCube m4"
          />
          <div
            className="labelCube m5"
          />
          <div
            className="labelCube m6"
          />
          <div
            className="labelCube m7"
          />
          <div
            className="labelCube m8"
          />
          <div
            className="labelCube m9"
          />
          <div
            className="labelCube m10"
          />
          <div
            className="labelCube m11"
          />
          <div
            className="labelCube m12"
          />
          <div
            className="labelCube m13"
          />
        </div>
         
      </div>
    </div>
    <div
      className="namer"
    >
      <div
        className="namer-input"
      >
        <input
          className=" roomInput"
          name="room"
          onChange={[Function]}
          placeholder="ROOM"
          type="text"
          value="my room"
        />
      </div>
    </div>
    <div
      className="labelTitle big"
    >
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube p1"
          />
          <div
            className="labelCube p2"
          />
          <div
            className="labelCube p3"
          />
          <div
            className="labelCube p4"
          />
          <div
            className="labelCube p5"
          />
          <div
            className="labelCube p6"
          />
          <div
            className="labelCube p7"
          />
          <div
            className="labelCube p8"
          />
          <div
            className="labelCube p9"
          />
          <div
            className="labelCube p10"
          />
          <div
            className="labelCube p11"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube l1"
          />
          <div
            className="labelCube l2"
          />
          <div
            className="labelCube l3"
          />
          <div
            className="labelCube l4"
          />
          <div
            className="labelCube l5"
          />
          <div
            className="labelCube l6"
          />
          <div
            className="labelCube l7"
          />
          <div
            className="labelCube l8"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube a1"
          />
          <div
            className="labelCube a2"
          />
          <div
            className="labelCube a3"
          />
          <div
            className="labelCube a4"
          />
          <div
            className="labelCube a5"
          />
          <div
            className="labelCube a6"
          />
          <div
            className="labelCube a7"
          />
          <div
            className="labelCube a8"
          />
          <div
            className="labelCube a9"
          />
          <div
            className="labelCube a10"
          />
          <div
            className="labelCube a11"
          />
          <div
            className="labelCube a12"
          />
          <div
            className="labelCube a13"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube y1"
          />
          <div
            className="labelCube y2"
          />
          <div
            className="labelCube y3"
          />
          <div
            className="labelCube y4"
          />
          <div
            className="labelCube y5"
          />
          <div
            className="labelCube y6"
          />
          <div
            className="labelCube y7"
          />
          <div
            className="labelCube y8"
          />
          <div
            className="labelCube y9"
          />
          <div
            className="labelCube y10"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube e1"
          />
          <div
            className="labelCube e2"
          />
          <div
            className="labelCube e3"
          />
          <div
            className="labelCube e4"
          />
          <div
            className="labelCube e5"
          />
          <div
            className="labelCube e6"
          />
          <div
            className="labelCube e7"
          />
          <div
            className="labelCube e8"
          />
          <div
            className="labelCube e9"
          />
          <div
            className="labelCube e10"
          />
          <div
            className="labelCube e11"
          />
          <div
            className="labelCube e12"
          />
          <div
            className="labelCube e13"
          />
        </div>
         
      </div>
      <div>
         
        <div
          className="labelLetterM"
        >
          <div
            className="labelCube r1"
          />
          <div
            className="labelCube r2"
          />
          <div
            className="labelCube r3"
          />
          <div
            className="labelCube r4"
          />
          <div
            className="labelCube r5"
          />
          <div
            className="labelCube r6"
          />
          <div
            className="labelCube r7"
          />
          <div
            className="labelCube r8"
          />
          <div
            className="labelCube r9"
          />
          <div
            className="labelCube r10"
          />
          <div
            className="labelCube r11"
          />
          <div
            className="labelCube r12"
          />
        </div>
         
      </div>
    </div>
    <div
      className="namer"
    >
      <div
        className="namer-input"
      >
        <input
          className=" playerInput"
          name="player"
          onChange={[Function]}
          placeholder="PLAYER"
          type="text"
          value="my name"
        />
      </div>
    </div>
    <button
      className="plus"
      disabled={false}
      type="submit"
    />
  </form>
</div>
`;

exports[`renders correctly with defaults 2`] = `
<div
  className="mainDivHome"
>
  <div
    className="redTetris "
  >
    <div
      className="box boxR"
    >
      <div
        className="boxCube boxR1"
      />
      <div
        className="boxCube boxR2"
      />
      <div
        className="boxCube boxR3"
      />
      <div
        className="boxCube boxR4"
      />
      <div
        className="boxCube boxR5"
      />
      <div
        className="boxCube boxR6"
      />
      <div
        className="boxCube boxR7"
      />
      <div
        className="boxCube boxR8"
      />
      <div
        className="boxCube boxR9"
      />
      <div
        className="boxCube boxR10"
      />
      <div
        className="boxCube boxR11"
      />
      <div
        className="boxCube boxR12"
      />
      <div
        className="boxCube boxR13"
      />
      <div
        className="boxCube boxR14"
      />
    </div>
    <div
      className="box boxE"
    >
      <div
        className="boxCube boxE1"
      />
      <div
        className="boxCube boxE2"
      />
      <div
        className="boxCube boxE3"
      />
      <div
        className="boxCube boxE4"
      />
      <div
        className="boxCube boxE5"
      />
      <div
        className="boxCube boxE6"
      />
      <div
        className="boxCube boxE7"
      />
      <div
        className="boxCube boxE8"
      />
      <div
        className="boxCube boxE9"
      />
      <div
        className="boxCube boxE10"
      />
      <div
        className="boxCube boxE11"
      />
      <div
        className="boxCube boxE12"
      />
      <div
        className="boxCube boxE13"
      />
    </div>
    <div
      className="box boxD"
    >
      <div
        className="boxCube boxD1"
      />
      <div
        className="boxCube boxD2"
      />
      <div
        className="boxCube boxD3"
      />
      <div
        className="boxCube boxD4"
      />
      <div
        className="boxCube boxD5"
      />
      <div
        className="boxCube boxD6"
      />
      <div
        className="boxCube boxD7"
      />
      <div
        className="boxCube boxD8"
      />
      <div
        className="boxCube boxD9"
      />
      <div
        className="boxCube boxD10"
      />
      <div
        className="boxCube boxD11"
      />
      <div
        className="boxCube boxD12"
      />
    </div>
    <div
      className="titleSpace"
    />
    <div
      className="boxs boxT"
    >
      <div
        className="boxCube boxT1"
      />
      <div
        className="boxCube boxT2"
      />
      <div
        className="boxCube boxT3"
      />
      <div
        className="boxCube boxT4"
      />
      <div
        className="boxCube boxT5"
      />
      <div
        className="boxCube boxT6"
      />
      <div
        className="boxCube boxT7"
      />
    </div>
    <div
      className="box boxE"
    >
      <div
        className="boxCube boxE1"
      />
      <div
        className="boxCube boxE2"
      />
      <div
        className="boxCube boxE3"
      />
      <div
        className="boxCube boxE4"
      />
      <div
        className="boxCube boxE5"
      />
      <div
        className="boxCube boxE6"
      />
      <div
        className="boxCube boxE7"
      />
      <div
        className="boxCube boxE8"
      />
      <div
        className="boxCube boxE9"
      />
      <div
        className="boxCube boxE10"
      />
      <div
        className="boxCube boxE11"
      />
      <div
        className="boxCube boxE12"
      />
      <div
        className="boxCube boxE13"
      />
    </div>
    <div
      className="boxs boxT"
    >
      <div
        className="boxCube boxT1"
      />
      <div
        className="boxCube boxT2"
      />
      <div
        className="boxCube boxT3"
      />
      <div
        className="boxCube boxT4"
      />
      <div
        className="boxCube boxT5"
      />
      <div
        className="boxCube boxT6"
      />
      <div
        className="boxCube boxT7"
      />
    </div>
    <div
      className="box boxR"
    >
      <div
        className="boxCube boxR1"
      />
      <div
        className="boxCube boxR2"
      />
      <div
        className="boxCube boxR3"
      />
      <div
        className="boxCube boxR4"
      />
      <div
        className="boxCube boxR5"
      />
      <div
        className="boxCube boxR6"
      />
      <div
        className="boxCube boxR7"
      />
      <div
        className="boxCube boxR8"
      />
      <div
        className="boxCube boxR9"
      />
      <div
        className="boxCube boxR10"
      />
      <div
        className="boxCube boxR11"
      />
      <div
        className="boxCube boxR12"
      />
      <div
        className="boxCube boxR13"
      />
      <div
        className="boxCube boxR14"
      />
    </div>
    <div
      className="boxs boxI"
    >
      <div
        className="boxCube boxI1"
      />
      <div
        className="boxCube boxI2"
      />
      <div
        className="boxCube boxI3"
      />
      <div
        className="boxCube boxI4"
      />
      <div
        className="boxCube boxI5"
      />
      <div
        className="boxCube boxI6"
      />
      <div
        className="boxCube boxI7"
      />
      <div
        className="boxCube boxI8"
      />
      <div
        className="boxCube boxI9"
      />
    </div>
    <div
      className="box boxS"
    >
      <div
        className="boxCube boxS1"
      />
      <div
        className="boxCube boxS2"
      />
      <div
        className="boxCube boxS3"
      />
      <div
        className="boxCube boxS4"
      />
      <div
        className="boxCube boxS5"
      />
      <div
        className="boxCube boxS6"
      />
      <div
        className="boxCube boxS7"
      />
      <div
        className="boxCube boxS8"
      />
      <div
        className="boxCube boxS9"
      />
      <div
        className="boxCube boxS10"
      />
      <div
        className="boxCube boxS11"
      />
      <div
        className="boxCube boxS12"
      />
      <div
        className="boxCube boxS13"
      />
      <div
        className="boxCube boxS14"
      />
    </div>
  </div>
  <form
    className="formFlex"
    onSubmit={[Function]}
  >
    <div
      className="namer"
    >
      <div
        className="namer-input"
      >
        <input
          className=" roomInput"
          name="room"
          onChange={[Function]}
          placeholder="ROOM"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="namer"
    >
      <div
        className="namer-input"
      >
        <input
          className=" playerInput"
          name="player"
          onChange={[Function]}
          placeholder="PLAYER"
          type="text"
          value=""
        />
      </div>
    </div>
    <button
      className="plus"
      disabled={true}
      type="submit"
    />
  </form>
</div>
`;
